tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -775,6 +777,15 @@ def get_rewriter_config(self):
     return get_tensorrt_rewriter_config(
         conversion_params=self._conversion_params)
 
+  def set_calib_tables(self, calib_tables):
+    """Set calibration tables into converted graph definition."""
+    assert self._converted, "Calibration tables can only be set on converted graph"
+    trt_nodes = [n for n in self._converted_graph_def.node if n.op == "TRTEngineOp"]
+    assert len(trt_nodes) == len(calib_tables), "Mismatch in number of TRT nodes and calibration tables"
+    for trt_node, calib_table in zip(trt_nodes, calib_tables):
```

Comment:
Please make `calib_table` a dict.

Reply:
`calib_table` is a serialized string. Doesn't seem like a good idea to pull out its members into key-val pairs.

Reply:
Sorry I meant a dict from node name (i.e. TRTEngineOp_i) to the value, instead of just values.